src/pages/Profile.js

```diff
@@ -355,22 +356,131 @@ const PassportLinker = ({ queryParams: { state, code } = {}, provider, prettyNam
 
 const sectionTitle = text => h2({ style: styles.sectionTitle }, [text])
 
-const Profile = ({ queryParams }) => {
-  // State
+const ExternalIdentitiesTab = ({ queryParams }) => {
+  return h(PageBox, { role: 'main', style: { flexGrow: 1 }, variant: PageBoxVariants.LIGHT }, [
+    h(NihLink, { nihToken: queryParams?.['nih-username-token'] }),
+    _.map(provider => h(FenceLink, { key: provider.key, provider }), allProviders),
+    !!getConfig().externalCredsUrlRoot && h(PassportLinker, { queryParams, provider: 'ras', prettyName: 'RAS' })
+  ])
+}
+
+const NotificationCheckbox = ({ notificationKeys, setSaving, prefsData }) => {
+  const notificationKeysWithValue = ({ notificationKeys, value }) => {
+    return Object.assign(...notificationKeys.map(notificationKey => ({ [notificationKey]: `${JSON.stringify(value)}` })))
+  }
+
+  return h(LabeledCheckbox, {
```

Comment:
These checkboxes are missing label text. It might be better to use `Checkbox` instead of `LabeledCheckbox` and add `aria-label`s. Alternatively, you could add visually hidden / screen reader only text for the label.

Reply:
Updated to pass the `label` text through to `aria-label` in 3d2fff105e421b1562be4fc820481a7ce8540cf7